modules/core/src/main/java/org/apache/fluo/recipes/core/types/TypedSnapshotBase.java

```diff
@@ -229,6 +229,17 @@ public ByteBuffer toByteBuffer(ByteBuffer defaultValue) {
       return toByteBuffer();
     }
 
+    public <T> T toObject(Class<T> clazz) {
```

Comment:
these changes were something I was experimenting with... did not intend to push... will delete later